table/get_context.cc

```diff
@@ -610,8 +610,13 @@ void replayGetContextLog(const Slice& replay_log, const Slice& user_key,
 
     (void)ret;
 
-    get_context->SaveValue(ikey, value, &dont_care, value_pinner);
+    Status read_status;
+    get_context->SaveValue(ikey, value, &dont_care, &read_status, value_pinner);
```

Comment:
Should we check the return value from SaveValue() ?

Reply:
I think we should check the return value as is done in the vicinity of this code.